Assignment3.txt

```diff
@@ -0,0 +1,11 @@
+Mariposa Lee 
+
+1a. o(n) = n. It will run until i is less than n
+b. o(n) = n+n^2. First, it will run n times, then n times for every n times (the for loops). 
+c. o(n) = n + log(n). First, it will run n times, then since theres an exponential function, the inverse of that would be log. 
```

Comment:
So close! since the loops are nested, they have to be multiplied rather than added! so this becomes O(nlogn)